src/applications/search/styles.scss

```diff
@@ -53,10 +54,26 @@
 
   hr {
     border-bottom: none;
+
+    &#hr-search-bottom {
+      margin-bottom: 1.5rem;
+    }
   }
 
   .results-footer {
     justify-content: space-between;
+    align-items: center;
+
+    .va-pagination {
+      width: unset;
+      border-top: none;
+
+      @media (min-width: 1201px) {
```

Comment:
@narin can we use any of the style variables here? How was 1201px determined?

https://github.com/department-of-veterans-affairs/design-system/blob/master/src/sass/base/_b-breakpoints.scss